.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+env:
+  GOPROXY: https://proxy.golang.org/
+  PROTOCOL_VERSION: "6.0"
+  PROTOCOL_VERSIONS: "6.0"
```

Comment:
ooo i almost overlooked this! ah so use of these protocol versions (6.0) assume that the provider is using `terraform-plugin-framework`  but i imagine the provider is still compatible w/protocol 5..so looking at the last TC build for the google provider, these params should probably be inline with

<img width="405" alt="Screen Shot 2021-11-18 at 12 23 28 AM" src="https://user-images.githubusercontent.com/13805595/142357032-08cf8977-c7fe-4abf-ae88-1973a9923631.png">

Reply:
yes, yes! will change, thanks!